src/inspector_agent.cc

```diff
@@ -219,8 +214,32 @@ bool RespondToGet(InspectorSocket* socket, const std::string& script_name_,
   return true;
 }
 
-}  // namespace
+std::string StringViewToUtf8(const StringView& view) {
+  if (view.is8Bit())
+    return std::string(reinterpret_cast<const char*>(view.characters8()),
+        view.length());
+
+  // Reserve 1 character for terminator.
+  std::string result(view.length() * sizeof(UChar) + 1, '\0');
```

Comment:
Thanks. I changed the sizeof & allowed the code to resize the buffer if need be.